.gitignore

```diff
@@ -4,18 +4,16 @@
 /public/css
 /public/storage
 /public/mix-manifest.json
+/results
 /storage/*.key
+/tests/Features/screenshots/
+/tests/Features/videos/
 /vendor
 .env
+.env.cypress.local
 .phpunit.result.cache
+.php_cs.cache
 Homestead.json
-.idea
 Homestead.yaml
 npm-debug.log
 yarn-error.log
-.php_cs.cache
-.DS_Store
```

Comment:
@asbiin Why did you remove that? 😢 

This files appears all the time on macos.

Reply:
It's a platform specific ignore, and should not be there.